bin/generic/root/index.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+const openapi = require('lib/openapi');
+const genericAction = require('bin/generic/action');
+const genericrResource = require('bin/generic');
+const genericUpdate = require('bin/generic/update');
+
+module.exports = (resource) => {
+    const name = resource.apiName || resource.name;
+    const discoveredCommands = [];
+
+    const paths = openapi.getResourcePaths(resource);
+
+    if (paths[`/${name}/{${name}Id}/actions/transfer`]) {
+        discoveredCommands.push('transfer');
```

Comment:
I think it is likely that in the future it could be moved to the generic definition of the generic command so that this command contains the logic to determinfe if generic command apply. I propose to keep it in this place until it forms.